packages/connect/src/protocol-connect/handler-factory.ts

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import type { MethodInfo } from "@bufbuild/protobuf";
+import { MethodIdempotency, protoBase64 } from "@bufbuild/protobuf";
+import type { JsonValue, MethodInfo } from "@bufbuild/protobuf";
 import { Message, MethodKind } from "@bufbuild/protobuf";
```

Comment:
The `import` (not `import type`) blocks should probably be merged here.

Reply:
Good catch, fixed.